README.md

```diff
@@ -1 +1,38 @@
-# Forms-for-Collecting-Data
\ No newline at end of file
+# Forms-for-Collecting-Data
+This project consists of building an HTML and CSS document that matches the appearance of the sing in form from  intuit webpage. the main objective of this project is to get the knowledge of how positioned the elements for web pages in this case a form for version intuit page.
+
+Additional description about the project and its features.
+
+Built With : Html,CSS and github technologies used:flex-box and grid of CSS Live Demo Linkhttps://rawcdn.githack.com/Fanger53/Forms-for-Collecting-Data/deaecc82d1bef1d7c37c38b9328756fd7a101cd4/index.html
+
+Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+Prerequisites: editor text, Html, CSS, github Setup Usage Deployment
+
+Authors
+
+👤 Author1
+
+Github: @BrianSammit
+
+Twitter:@CruzSammit
```

Comment:
- [ ] [OPTIONAL] Github and twitter handles should be actual links to facilitate easy networking with potential collaborators and/or recruiters.